pytorch_to_returnn/torch/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .nn.functional import *
 from .serialization import load
 from .autograd import *
-from .jit import *
-from .onnx import *
 from . import nn
+import torch.cuda
+import torch.onnx
+import torch.jit
```

Comment:
```suggestion
from . import cuda
from . import onnx
from . import jit
```
But double check that this is also done in PyTorch. Otherwise it should be left away.